packages/create-redwood-app/template/api/db/seed.js

```diff
@@ -5,38 +5,33 @@ const dotenv = require('dotenv')
 dotenv.config()
 const db = new PrismaClient()
 
+/*
+* Seed data is database data that needs to exist for your app to run.
+*
+* @see https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-reset
+* @see https://www.prisma.io/docs/guides/prisma-guides/seed-database
+* @see https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#upsert
+* @see https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany
+*/
```

Comment:
just pasted this file in a fresh app and eslint/prettier had me insert these spaces; thought this would be caught by CI but maybe doesn't apply to the crwa template?
```suggestion
/*
 * Seed data is database data that needs to exist for your app to run.
 *
 * @see https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-reset
 * @see https://www.prisma.io/docs/guides/prisma-guides/seed-database
 * @see https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#upsert
 * @see https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany
 */
```